js/index.js

```diff
@@ -142,24 +142,43 @@ function hideShow () {
 //function call to hide or show message list when page loads
 hideShow();
 
-let gitHubRequest = new XMLHttpRequest();
 
-gitHubRequest.open("GET", "https://api.github.com/users/nicoleCodeGirl/repos", true);
-gitHubRequest.send();
 
-gitHubRequest.onload = function() {
-    let repositories = JSON.parse(this.response);
-    console.log(repositories);
+// let gitHubRequest = new XMLHttpRequest();
+
+// gitHubRequest.open("GET", "https://api.github.com/users/nicoleCodeGirl/repos", true);
+// gitHubRequest.send();
+
+// gitHubRequest.onload = function() {
+//     let repositories = JSON.parse(this.response);
+//     console.log(repositories);
```

Comment:
When you don't need code any longer, better to go ahead and delete it to keep things tidy.